src/main/java/com/github/monster860/fastdmm/objtree/ObjectTree.java

```diff
@@ -468,4 +475,24 @@ public void valueForPathChanged(TreePath arg0, Object arg1) {
 	public String toString() {
 		return dmePath;
 	}
+
+	/*
+	 * Gets a file name, taking the project's FILE_DIR into account.
+	 * @param fileName The relative name of the file, which should be found with FILE_DIR definitions.
+	 * @return The relative file path from the root project folder (folder with the opened code file).
+	 * @exception FileNotFoundException Thrown if the file name could not be resolved.
+	 */
+	public String filePath(String filePath) throws FileNotFoundException {
+		for(Iterator<Path> i = fileDirs.iterator(); i.hasNext();) {
```

Comment:
There is no need to use `iterator()` here, doing `for (Thing t : things) {}` would be cleaner